content/en/tools/digital-currencies/glossaryitems.json

```diff
@@ -0,0 +1,52 @@
+{
+	"CBDC": {
+		"definition": "Central bank digital currency: a central bank-issued digital money, denominated in a national unit of account, which is backed by the full faith and credit of the issuing central bank.",
+		"more": ""
+	},
+	"custodian": {
+		"definition": "EA custodian attests to the reception of the underlying collateral funds for tokens created by the TIE on the network. It verifies collateral payments that are received from a network participant through banking deposits either held with a central bank or a commercial bank.",
+		"more": "For more information, see the topic <a href=\"./roles-overview.html\">Roles Overview</a>" 
+	},
+	"issuance": {
+		"definition": "The process where a central bank or commercial bank creates new tokens and then transfers them to another party.",
+		"more": "For more information, see the topic <a href=\"./issuing-tokens.md\">Issuing Tokens</a>."
+	},
+
+    "redemption": {
+        "definition": "When a commercial bank or central bank removes a token, which it defined, from a transacting entity or a network.",
+        "more": ""
```

Comment:
Not sure if it's on purpose but "more" link is missing in some of the definitions in this file.

Reply:
This is on purpose for the moment.